content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -217,6 +226,20 @@ This reference manager also integrates nicely with other word processing softwar
 
 ![](insert-citation.png)
 
+## Citing R and R packages
+
+The `knitr::write_bib()` function comes in handy if you need to cite R packages.
+The first argument accepts a character vector of R package names and the second argument can be used to specify a `packages.bib` file to which the BibTex entries will be written.
+
+For instance:
+
+```{r}
+# to cite specific packages
+knitr::write_bib(c("base", "bookdown", "rmarkdown"), "packages.bib")
+```
```

Comment:
Will this be executed each time the tutorial Rmd is knitted? I'm not sure but this could mean that duplicate entries would be added each time, to the bib file.

Reply:
No duplicates are generated when hitting the knit button again. So it's OK